test/test_http_proxy.rb

```diff
@@ -0,0 +1,41 @@
+require 'test_helper'
+
+# Http proxy test
+class TestHttpProxy < MiniTest::Test
+  def setup
+    @path = 'http://localhost:8080/api/v1/proxy/namespaces/default/services/image-scanner:5001-tcp/image-scanner/api/scan'
+    @client = Kubeclient::Client.new 'http://localhost:8080/api/', 'v1',
+                                     auth_options: { bearer_token: 'valid_token' }
+    @proxy = @client.proxy_url_service('default', 'image-scanner', '5001-tcp')
+  end
+
+  def test_instance_type
+    assert_instance_of(Kubeclient::Common::HttpProxyWrapper, @proxy)
+  end
+
+  def test_post_from_json_v3
+    stub_request(:post, @path)
+      .with(headers: { Authorization: 'Bearer valid_token' },
+            body: '{"number" => 2, "scan" => ["docker.io/richxsl/rhel7"]}')
+      .to_return(body: open_test_json_file('http_proxy.json'),
+                 status: 200)
+
+    @proxy.post('/image-scanner/api/scan',
+                '{"number" => 2, "scan" => ["docker.io/richxsl/rhel7"]}')
+    assert_requested(:post,
+                     @path,
+                     times: 1)
+  end
+
+  def test_get_from_json_v3
```

Comment:
The name of the method is not descriptive of what the test does (should be related to proxy).